packages/protobuf-bench/package-lock.json

```diff
@@ -0,0 +1,564 @@
+{
+  "name": "@bufbuild/protobuf-bench",
```

Comment:
This lock file should not exists. In a project with npm workspaces, there must only be a single lock file.

Reply:
I see. That makes sense. Thank you. Removed both lock files in protobuf-bench and protobuf-example.